wp-job-manager-functions.php

```diff
@@ -66,7 +66,10 @@ function get_job_listings( $args = array() ) {
 		}
 
 		if ( ! empty( $args['search_location'] ) ) {
-			$location_meta_keys = array( 'geolocation_formatted_address', '_job_location', 'geolocation_state_long' );
+			if ( 'remote' === $args['search_location'] || 'Remote' === $args['search_location'] ) {
```

Comment:
If we were to use this natural language search, we'd probably want to let these be translatable.

Reply:
@0xDELS or @jom can u fix this and send it to the next update? This is still needed highly due to covid.

Reply:
> We are looking for this feature to be added to the plugin as it has become really important for SEO - has there been an update on this or what the conflicts were/are?

An update for this is in the works.